package.json

```diff
@@ -22,7 +22,7 @@
     "express": "^4.14.0",
     "express-session": "^1.15.0",
     "lru-native": "^0.4.0",
-    "message-factory": "21-23/message-factory#v2.8.0",
+    "message-factory": "21-23/message-factory#v3.0.0",
```

Comment:
Yes, it's just a matter of merging https://github.com/21-23/front-end-service/pull/15. But that shows in which version of MF the messages were changed